project/interpolation.go

```diff
@@ -8,6 +8,17 @@ import (
 	"github.com/Sirupsen/logrus"
 )
 
+func isNum(c uint8) bool {
+	return c >= '0' && c <= '9'
+}
```

Comment:
It still kind of surprises me that there is no golang builtin for super common operations like this